packages/button/src/Button/components/GenericButton.spec.tsx

```diff
@@ -86,3 +86,48 @@ describe("<GenericButton />", () => {
     expect(screen.getByTestId("custom-prop")).toBeInTheDocument()
   })
 })
+
+describe("<GenericButton /> with native HTML `form` attributes", () => {
+  const buttonFormAttributes = {
+    form: "linked-form-id",
+    formAction: "/",
+    formMethod: "post",
+    formEncType: "text/plain",
+    formTarget: "_self",
+    formNoValidate: false,
+  } satisfies ButtonFormAttributes
+
+  it("will pass form props into the button", () => {
+    render(
+      <GenericButton
+        label="submit button"
+        data-testid="custom-prop"
+        {...buttonFormAttributes}
+      />
+    )
+
+    expect(screen.getByRole("button", { name: "submit button" })).toBeTruthy()
```

Comment:
I think you can clean this up a little:
- We should use `toBeInTheDocument` here, rather than `toBeTruthy` which can have weird side effects (e.g. imagine a mistake in your expectation that returns `1` - that would be "truthy" but maybe not what you actually wanted to test)
- Having made the above point, you don't actually need the first `expect`, because the 2nd one - where you assert the `form` attribute will fail if the button can't be found

Reply:
True, will just remove the first one to clean up the test